polaris-react/src/styles/_tokens.scss

```diff
@@ -0,0 +1,32 @@
+/* stylelint-disable unit-disallowed-list */
+/* stylelint-disable length-zero-no-unit */
+
+/* Breakpoints - Aliases */
+$p-breakpoints-xs: 0px;
+$p-breakpoints-xs-em: 0em;
+
+$p-breakpoints-sm: 480px;
+$p-breakpoints-sm-em: 30em;
+
+$p-breakpoints-md: 768px;
+$p-breakpoints-md-em: 48em;
+
+$p-breakpoints-lg: 1008px;
+$p-breakpoints-lg-em: 63em;
```

Comment:
You might be wondering why I have included both `px` and `em` breakpoint aliases.. My thinking was that users may need to compose breakpoints (such as: `$between-xs-and-md` or `$mobile-nav-collapsed`) and these variables would serve as the low level building blocks. Thoughts?

Reply:
Sounds good, I'll remove the `px` aliases. We really only need the `em` aliases for the above note about composition.

Reply:
Updated: https://github.com/Shopify/polaris/pull/5558/commits/e6642873d983fd6694bec4091d3e2be816c2cfc3

<details>
  <summary>Example - Generated output</summary>
  
 ```scss
 /* stylelint-disable unit-disallowed-list */
/* stylelint-disable length-zero-no-unit */

/* Breakpoints - Aliases */
$p-breakpoints-xs: 0em;
$p-breakpoints-sm: 30em;
$p-breakpoints-md: 48em;
$p-breakpoints-lg: 63em;

/* Breakpoints - Media conditions */
$p-breakpoints-xs-up: '(min-width: 0em)';
$p-breakpoints-xs-down: '(max-width: 0em)';
$p-breakpoints-xs-only: '(min-width: 0em) and (max-width: 30em)';

$p-breakpoints-sm-up: '(min-width: 30em)';
$p-breakpoints-sm-down: '(max-width: 30em)';
$p-breakpoints-sm-only: '(min-width: 30em) and (max-width: 48em)';

$p-breakpoints-md-up: '(min-width: 48em)';
$p-breakpoints-md-down: '(max-width: 48em)';
$p-breakpoints-md-only: '(min-width: 48em) and (max-width: 63em)';

$p-breakpoints-lg-up: '(min-width: 63em)';
$p-breakpoints-lg-down: '(max-width: 63em)';
$p-breakpoints-lg-only: '(min-width: 63em)';
 ```
</details>